oci-test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -ex
+
+export FN_TEST_MODE="OCI"
+
+# FILL IN VALUES BELOW. These values should match with values in the following files:
+# oci-auth/fn/config.yaml
+# oci-auth/fn/contexts/functions-test.yaml
+# oci-auth/oci/config
+# simpleapp/app.json
+export FN_API_URL="https://functions.us-ashburn-1.oraclecloud.com/20181201"
```

Comment:
This domain should be `functions.us-ashburn-1.oci.oraclecloud.com`

Reply:
Will ask the user to fill this in.